examples/MQ/pixelDetector/macros/run_sim.C

```diff
@@ -49,6 +49,7 @@ void run_sim(Int_t nEvents = 10, TString mcEngine = "TGeant3", Int_t fileId = 0,
     // -----   Create simulation run   ----------------------------------------
     FairRunSim* run = new FairRunSim();
     run->SetName(mcEngine);   // Transport engine
+    run->SetSimulationConfig(new FairVMCConfig());
```

Comment:
The `FairVMCConfig` object appears to be leaking (not going to repeat this comment for the other places in this PR).